app/assets/scripts/views/owner.js

```diff
@@ -84,17 +76,20 @@ var Owner = React.createClass({
               </div>
               <div className='inpage__col--content'>
                 <ul className='inpage-stats'>
-                  <li> {shortTally(totalBudget)} <small>Total Funds</small></li>
-                  <li> {tally(ownerProjects.length)} <small>{singleProject ? 'Project' : 'Projects'} Funded</small></li>
+                  <li> {activeProjects} <small>Active {activeProjects > 1 ? 'Projects' : 'Project'}</small></li>
+                  <li> {ownerProjects.length} <small>Total {ownerProjects.length > 1 ? 'Projects' : 'Project'}</small></li>
                 </ul>
                 <div className='inpage__overview-chart'>
-                  {!singleProject && (<HorizontalBarChart
-                    lang={this.props.meta.lang}
-                    data={chartData}
-                    margin={{ left: 300, right: 50, top: 10, bottom: 50 }}
-                    xFormat={shortTally}
-                    yFormat={shortText}
-                  />)}
+                  <div className='chart-content'>
+                    <h3>Number Served</h3>
```

Comment:
If `singleProject` is true then you'll just get this heading and no chart below.